### DISCUSSION TITLE
Access bitmap gallery via API?

### DISCUSSION THREAD

**Discussion by darrenparkinson** [pixelit-project/PixelIt#251] [Q&A] [answered]
In the documentation under Pixel Gallery, it says:

> *The Pixel Gallery is the collection point of the BMP (PixelArt images), the BMPs that are in the Pixel Gallery are also accessible via the API.*

This suggests that it's possible to access the Pixel Gallery via API, but I can't seem to find any reference to it?

Any thoughts greatly appreciated.
Many thanks.

**Comment by foorschtbar** [comment, marked as answer]
Yes, there is no documentation, but you could look in the sourcecode. Thats the API entpoint for getting the images: `https://pixelit.bastelbunker.de/API/GetBMPByID/<ID>`

**Reply by darrenparkinson** [reply]
Ah, yes, that was going to be my next port of call -- I thought I'd need to look in the node red code? You can't send the ID to the pixelit itself can you?

**Reply by o0shojo0o** [reply]
No it was meant at that time not to need external connections.

**Comment by o0shojo0o** [comment]
If you want to use the API for something else like the PixelIt please contact me first.

**Reply by darrenparkinson** [reply]
Oh, of course, no this is absolutely for the PixelIt. Thinking of making a CLI for it -- not a huge Node Red fan I'm afraid...

**Comment by o0shojo0o** [comment]
Interesting endpoints would be for you (all reachable as HTTP GET request):
`/api/GetBMPByID/<id>`
`/api/GetBMPNewst`
`/api/Releases`
